js/data/locale/ceb/dateformats.json

```diff
@@ -56,72 +50,250 @@
                 "ah": "HH",
                 "hm": "HH:mm",
                 "ms": "mm:ss",
-                "ahm": "HH:mm",
-                "hms": "HH:mm:ss",
-                "hmz": "HH:mm z",
-                "ahmz": "HH:mm z",
-                "ahms": "HH:mm:ss",
-                "hmsz": "HH:mm:ss z",
-                "ahmsz": "HH:mm:ss z"
+                "h": "HH",
+                "m": "mm",
+                "s": "ss"
             }
         },
         "range": {
             "c00": {
-                "f": "{sm} {sd}, {sy} sa {st} – {et}",
+                "s": "{sm}/{sd}/{sy}, {st} – {et}",
+                "m": "{sm} {sd}, {sy}, {st} – {et}",
                 "l": "{sm} {sd}, {sy} sa {st} – {et}",
+                "f": "{sm} {sd}, {sy} sa {st} – {et}"
+            },
+            "c01": {
+                "s": "{sm}/{sd}/{sy}, {st} – {em}/{ed}/{ey}, {et}",
+                "m": "{sm} {sd}, {sy}, {st} – {em} {ed}, {ey}, {et}",
+                "l": "{sm} {sd}, {sy} sa {st} – {em} {ed}, {ey} sa {et}",
+                "f": "{sm} {sd}, {sy} sa {st} – {em} {ed}, {ey} sa {et}"
+            },
+            "c02": {
+                "s": "{sm}/{sd}/{sy}, {st} – {em}/{ed}/{ey}, {et}",
+                "m": "{sm} {sd}, {sy}, {st} – {em} {ed}, {ey}, {et}",
+                "l": "{sm} {sd}, {sy} sa {st} – {em} {ed}, {ey} sa {et}",
+                "f": "{sm} {sd}, {sy} sa {st} – {em} {ed}, {ey} sa {et}"
+            },
+            "c03": {
+                "s": "{sm}/{sd}/{sy}, {st} – {em}/{ed}/{ey}, {et}",
+                "m": "{sm} {sd}, {sy}, {st} – {em} {ed}, {ey}, {et}",
+                "l": "{sm} {sd}, {sy} sa {st} – {em} {ed}, {ey} sa {et}",
+                "f": "{sm} {sd}, {sy} sa {st} – {em} {ed}, {ey} sa {et}"
+            },
+            "c10": {
+                "s": "{sm}/{sd}/{sy} – {em}/{ed}/{ey}",
+                "m": "{sm} {sd} – {ed}, {ey}",
+                "l": "{sm} {sd} – {ed}, {ey}",
+                "f": "{sm} {sd} – {ed}, {ey}"
+            },
+            "c11": {
+                "s": "{sm}/{sd} – {em}/{ed}/{ey}",
+                "m": "{sm} {sd} – {em} {ed}, {ey}",
+                "l": "{sm} {sd} – {em} {ed}, {ey}",
+                "f": "{sm} {sd} – {em} {ed}, {ey}"
+            },
+            "c12": {
+                "s": "{sm}/{sd}/{sy} – {em}/{ed}/{ey}",
+                "m": "{sm} {sd}, {sy} – {em} {ed}, {ey}",
+                "l": "{sm} {sd}, {sy} – {em} {ed}, {ey}",
+                "f": "{sm} {sd}, {sy} – {em} {ed}, {ey}"
+            },
+            "c20": {
+                "s": "{sm}/{sy} – {em}/{ey}",
+                "m": "{sm}, {sy} – {em}, {ey}",
+                "l": "{sm}, {sy} – {em}, {ey}",
+                "f": "{sm}, {sy} – {em}, {ey}"
+            },
+            "c30": {
+                "f": "{sy} – {ey}",
+                "l": "{sy} – {ey}",
+                "m": "{sy} – {ey}",
+                "s": "{sy} – {ey}"
+            }
+        }
+    },
+    "islamic": "gregorian",
```

Comment:
many `dateformat.json` files have these part. line from 166. 
It's duplicated and I think enough to have in root.

Reply:
Ran `mergeAndPrune` on all the date formats data